.husky/pre-commit

```diff
@@ -1,2 +1,3 @@
+npm run test:svg
```

Comment:
In case of a broken svg, this runs and prints the error message, but still continues executing the script and finishes the commit. If we change this to
`npm run test:svg || exit 1`
then the husky hook will actually prevent the commit. Don't know if that's what we want. Haven't worked with anything like husky before.

Reply:
husky is just tooling allowing this pre commit hook to easily be added and managed between multiple people / systems.
the code is actually just a regular git pre commit hook that is run.

I'll merge it as it is now to get it in for CI, but I'm open to having exit 1 for the pre commit hook! I'll open another PR